docs/network-policy.md

```diff
@@ -1,4 +1,50 @@
-# Cluster Network Policy
+# Antrea Network Policies
+
+## Summary
+
+Antrea supports standard K8s NetworkPolicies to secure traffic between Pods. These
+NetworkPolicies are written from an application developers perspective, hence they
+lack the ability to gain a finer-grained control over the security policies that
+a cluster administrator would require. This document describes a few new CRDs
+supported by Antrea to provide the administrator with more control over
+security within the cluster and which are meant to co-exist with and complement the
```

Comment:
nit: s/within the cluster and which are meant/within the cluster, and which are meant